# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by scripts/Create-CopyPatch.
# 
# T2 SDE: package/*/dhcp/mipsel.patch.cross
# Copyright (C) 2015 - 2021 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

Now, this is a little bit of a quick & dirty hack. Somehow the CC_FOR_BUILD is not
set to our cross compiler, and thus the __mipsel__ check woudl not work.

Simply hardcode "mips" for now, in the hope to get away with it.

--- dhcp-4.4.2/bind/bind-9.11.14/config.guess.vanilla	2021-01-18 11:47:13.308974042 +0100
+++ dhcp-4.4.2/bind/bind-9.11.14/config.guess	2021-01-18 11:47:33.970975107 +0100
@@ -968,6 +968,7 @@
 EOF
 	eval `$CC_FOR_BUILD -E $dummy.c 2>/dev/null | grep '^CPU'`
 	test x"${CPU}" != x && { echo "${CPU}-unknown-linux-${LIBC}"; exit; }
+	echo "${UNAME_MACHINE}-unknown-linux-gnu" && exit 0
 	;;
     or1k:Linux:*:*)
 	echo ${UNAME_MACHINE}-unknown-linux-${LIBC}
